src/app/containers/ConsentBanner/CanonicalLogic/index.js

```diff
@@ -15,7 +15,7 @@ const PRIVACY_COOKIE_PREVIOUS_VALUES = ['0', '1'];
 const onClient = typeof window !== 'undefined';
 
 const setPolicyCookie = (value, logger) => {
-  setCookie(POLICY_COOKIE, value);
+  setCookie({ name: POLICY_COOKIE, value, sameSite: 'None' });
```

Comment:
I'm going to leave some comments for my own benefit/understanding rather than as feedback necessarily!
These three cookies are set to sameSite 'None' because they can be accessed and modified on `.co.uk` as well as `.com`

Reply:
Think an inline comment mentioning that we are trying to maintain consistency with Orbit/cross-TLD browsing would help future debugging.